src/main/java/com/google/sps/servlets/MapMarker.java

```diff
@@ -89,50 +41,27 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
 
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
 
-    // To store new marker.
-    if (newMarker.getId() == -1) {
```

Comment:
I left that part because I thought that later (when user would want to preview already saved route) user should be able to update information about a marker.  I'm not sure how to do it instead.